<launch>

    <!-- DOPE to object_pose_msgs converter-->
    <include file="$(find pose_selector)/launch/dope_converter.launch"/>

    <!-- Pose selector -->
    <!-- Set pose_selector subscriber to dope converter output -->
    <include file="$(find pose_selector)/launch/pose_selector.launch">
        <arg name="subscribe_topic" value="/dope_converter_poses"/>
    </include>

</launch>